packages/server/lib/browsers/chrome.ts

```
@@ -146,6 +154,11 @@ const _normalizeArgExtensions = function (extPath, args, pluginExtensions, brows
   })
 
   if (loadExtension) {
+    // @see https://github.com/cypress-io/cypress/issues/31702
+    if (Number(browser.majorVersion) >= 137 && browser.name === 'chrome') {
+      throw new ExtensionLoadError()
+    }
+
```

@AtofStryker So I believe this is only handling the case when a user manually pushes `--load-extension` into the browser options via launchOptions.args, this is not handling the case of someone using our documented API on how to load extensions via `launchOptions.extensions.push` https://docs.cypress.io/api/node-events/browser-launch-api#Add-browser-extensions

---

should be

```
if (userExtensions.length > 0) {
    // @see https://github.com/cypress-io/cypress/issues/31702
    if (Number(browser.majorVersion) >= 137 && browser.name === 'chrome') {
      throw new ExtensionLoadError()
    }
}
```

on line 171

remove 157-161 new changes

---

It is handling the `launchOptions.extensions` as that is what gets passed in to the function https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/browsers/chrome.ts#L570. I need to add/update the CI jobs to make sure both scenarios are covered, it's just a bit challenging since its a beta channel

---

should now be ready to go. Errors get swallowed in open mode that throw when launching the browser so I had to opt for a warning

---

case covers both `--load-extension` and `launchOptions.extensions`